roles/openshift_node/defaults/main.yml

```diff
@@ -75,6 +75,7 @@ openshift_node_support_packages_base:
   - bash-completion
   - vim-minimal
   - nano
+  - openvswitch
```

Comment:
@mccv1r0 now that I look back at it, we need openvswitch2.13 as the package name.